pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -258,7 +275,7 @@ func (o *OvirtMapper) MapDataVolumes(targetVMName *string) (map[string]cdiv1.Dat
 				},
 				PVC: &corev1.PersistentVolumeClaimSpec{
 					AccessModes: []corev1.PersistentVolumeAccessMode{
-						corev1.ReadWriteOnce,
+						accessMode,
```

Comment:
will this require changes to the functional tests setup to make sure PVs with proper access modes are available or the default PVs supports all types of access-modes ?

Reply:
Actually, you can create `ReadWriteMany` PVC even on storageclass, which doesn't support RWX acccess mode. So it doesn't but I will add `rook-ceph` in other PR, so we have real test case, and also we may test multiple storage classes.